spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,46 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response(app_id: nil, platform: nil, review_id: nil, response: "")
+      data = {
+        responseText: response,
+        reviewId: review_id
+      }
+      r = request(:post) do |req|
+        req.url "ra/apps/#{app_id}/platforms/#{platform}/reviews/#{review_id}/responses"
+        req.body = data.to_json
+        req.headers['Content-Type'] = 'application/json'
+      end
+      parse_response(r, 'data')
+    end
+
+    def update_developer_response(app_id: nil, platform: "ios", review_id: nil, response_id: nil, response: "")
+      data = {
+        responseText: response
+      }
+      r = request(:put) do |req|
+        req.url "ra/apps/#{app_id}/platforms/#{platform}/reviews/#{review_id}/responses/#{response_id}"
+        req.body = data.to_json
+        req.headers['Content-Type'] = 'application/json'
+      end
+      parse_response(r, 'data')
+    end
+
     def get_reviews(app_id, platform, storefront, versionId = '')
-      r = request(:get, "ra/apps/#{app_id}/reviews?platform=#{platform}&storefront=#{storefront}&versionId=#{versionId}")
-      parse_response(r, 'data')['reviews']
+      index = 0
+      per_page = 100 # apple default
+      all_fetched = false
```

Comment:
You can get rid of this variable to reduce side-effects.
It is better to `break` when the loop condition is filled.